prow/gerrit/adapter/adapter.go

```diff
@@ -280,8 +283,27 @@ func (c *Controller) ProcessChange(instance string, change client.ChangeInfo) er
 	case client.New:
 		presubmits := c.config().Presubmits[cloneURI.String()]
 		presubmits = append(presubmits, c.config().Presubmits[cloneURI.Host+"/"+cloneURI.Path]...)
+
 		var filters []pjutil.Filter
-		filter, err := messageFilter(c.lastUpdate, change, presubmits)
+		var latestReport *reporter.JobReport
+		account := c.gc.Account(instance)
+		if account == nil {
```

Comment:
we probably still want to support anonymous accounts, just skip parsing comments instead of abort?

Reply:
this is the prow account, this shouldn't really happen since that means auth failed. Added a comment regarding that.